index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500&display=swap" rel="stylesheet">
+    <title>Portofolio</title>
+</head>
+<body>
+    <header>
+        <div class="container">
+        <a class="logo">Saba Ahmad</a>
+          <nav id="items">
+        <i class="fa fa-bars"></i>
+     </nav>
+     </div>
+</header>
+<section>
+<div class="content">
+    <h1>I'm Saba <br> Glad to see <br> you!</h1>
```

Comment:
The previous reviewer mentioned that you give reasons when suggesting changes, this will help the author of the project understand the reasons to make the changes you have suggested 😄 😄
- #### Kindly update your comment with an appropriate reason for the suggested changes.